build/bazel/remote/asset/v1/remote_asset.proto

```diff
@@ -201,6 +201,11 @@ message FetchBlobRequest {
   //
   // Specified qualifier names *MUST* be unique.
   repeated Qualifier qualifiers = 5;
+
+  // The digest function the server must use to compute the digest.
+  //
+  // If unset, the server SHOULD default to SHA256.
```

Comment:
I do agree with you this is a little strict, and I wrote SHOULD because that is the default Bazel behavior today. It does not seem like a good idea to me to split the source of truth about which digest function to use across both this field and the qualifier field.

We could relax this command and say it's up to the server here though?

Reply:
> If unset, the server SHOULD select one of MD5, MURMUR3, SHA1, SHA256, SHA384, SHA512, or VSO, based on a server-side default value.

Reply:
I am recreating this PR in https://github.com/bazelbuild/remote-apis/pull/286 with @EdSchouten suggestion added.